### ISSUE TITLE
Unusable application backup as it tries to install a version of ffmpeg that is not available

### ISSUE THREAD

**Issue by ebouaziz** [YunoHost-Apps/jellyfin_ynh#98]
Hi,

### Describe the bug

Backup restoration fails because it depends on a jellyfin-ffmpeg package that is not available anymore.

This was experienced after a failed upgrade, which is another matter.

### Context

- Hardware: LXC container on x86_64
- YunoHost version: 4.3.6.3
- I have access to my server: Through SSH | through the webadmin | direct access via keyboard / screen | ...
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: yes
  - If yes, please explain: added geo filter in nginx to restrict access by country
- Using, or trying to install package version/branch: jellyfin 10.7.7~yhn2

### Steps to reproduce

    ```sh
    sudo yunohost backup restore jellyfin-pre-upgrade1
    ```

### Expected behavior

App is restored

### Logs

The restore operation fails:

https://paste.yunohost.org/raw/oqetujazoj

Long story short: the file apps/jellyfin/settings/conf/ffmpeg.*.src points to a ffmpeg package that is not online anymore.

I extracted the archive, updated the version to 5.0.1-6 and the hash, rebuilt the archive, restarted the restore operation and it succeded.

regards.

**Comment by Salamandar** [comment]
Linked to https://github.com/YunoHost-Apps/jellyfin_ynh/issues/96

**Comment by arthurlutz** [comment]
Hitting this bug too. Am looking for a way to modify the tar backup to modify this URL to introduce the old URL... if you have a workaround, I'll take it!

**Comment by arthurlutz** [comment]
Asked on the matrix forum and got the following link : https://mirrors.sdu.edu.cn/spark-store-repository/store/depends/jellyfin/ For some reason the sha256sum is different. Trying to re-create the tar with patched values. 

Is there a way to install the new version and restore only the data from the tar ?

**Comment by tituspijean** [comment]
You can definitely patch the URL in the tar file. (Untar and retar, or patch in place, there are multiple ways to do so on Debian).

Sorry for the inconvenience in any case, since it has been 8 months, I am closing. Feel free to reopen if needed.
